### PR TITLE
Content Moderator SDK initial files

### PR DESCRIPTION
**Pull request by v-sodsou**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Helpers/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,1296 @@
...
+		[Description(@"[
+			{
+				""TimeStamp"": ""123"",
+
+				""Terms"":
+
+				[
+					{
+						""Index"": 0,
+						""Term"" : ""fuck""
```
Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Helpers/Constants.cs]: @v-sodsou can you clarify what is this all about?
With this, you will be blocked at sign/publish time when we run bunch of static analysis tools.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by v-sodsou** [comment]
@shahabhijeet 
Hi Abhijeet, 
Thanks for the review . I have updated the files as per your comments.

-Sonali Dsouza

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <Content Include="TestDataSources\NormalImages\TiffOCR.tiff">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Net.Http" Version="4.3.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\ContentModerator\Microsoft.Azure.CognitiveServices.ContentModerator.csproj" />
+  </ItemGroup>
```
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj]: Removed the mentioned lines.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Data/Response.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Rest;
+using Microsoft.CognitiveServices.ContentModerator.Models;
+using Microsoft.CognitiveServices.ContentModerator;
+
+namespace CMTests
```
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Data/Response.cs]: Changed the namespace.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Data/ReviewData.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CMTests.Data
+{
+    public class ReviewData
+    {
+        public ReviewData()
+        {
```
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Data/ReviewData.cs]: Removed all the line breaks.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Data/ReviewResponse.cs [hunk 5]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Rest;
+using Microsoft.CognitiveServices.ContentModerator.Models;
+using Microsoft.CognitiveServices.ContentModerator;
+
+namespace CMTests
```
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Data/ReviewResponse.cs]: Changed the namespace.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Data/ReviewResponse.cs [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+{
+   public class ReviewResponses
+    {
+        public ReviewResponses()
+        {
+
```
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Data/ReviewResponse.cs]: Removed the line break.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Helpers/Constants.cs (hunk 1 above)
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/Helpers/Constants.cs]: This was test data from our test team. Modified the test data.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/ImageModerator.cs [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+
+        
+        public void TestCleanup()
+        {
+            TestCleanupConfiguration();
+        }
+
+        [Fact]
+        [assembly: Collection("BVT")]
+        [assembly: CollectionBehavior(DisableTestParallelization = true)]
```
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/ImageModerator.cs]: Removed assembly collection.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj [hunk 8]
```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+ <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
```
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj]: Changed it to above line.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.ContentModerator</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Content Moderator APIs.</Description>
+    <Version>1.0.0-preview</Version>
```
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj]: Changed the version to 0.9

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj (hunk 2 above)
[the remark this answers is not in the document]
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj]: Modified as per review comment.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+    <None Update="TestDataSources\NormalImages*.*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
```
Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj]: @v-sodsou please remove line 23 to 25

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj (hunk 10 above)
Comment (v-sodsou) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator.Tests/CognitiveServices.ContentModerator.Tests.csproj]: @shahabhijeet  System.Net.Http is being used in test project.It gives me a build error if I remove it.

**Pull request merged**